teams/3/list-examples.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="./styles.css" />
+        <link rel="stylesheet" href="../../css/normalize.css" />
+        <link rel="stylesheet" href="../../css/main.css" />
+        <link rel="stylesheet" href="../../css/vertical-navigation.css" />
+        <title>Document</title>
+    </head>
+    <body>
+        <h1>List examples</h1>
+        <section>
+            <h2>Example number 1</h2>
+            <ul class="ut-list-cols-2">
+                <li class="text-xl-med">
+                    <img src="./assets/clock.svg" alt="clock" />
+                    <h3>Access to daily analytics</h3>
```

Comment:
Why are these elements h3, this should be span. Remember that this are examples to copy and paste.

Reply:
This list is in our team's Figma. This is h3 because it is the subheading for the Features list. So the Feature is heading 2 and then it has a list of features and each feature has its own title as well as a description of what does it mean, that's why we decided to go with h3.

Reply:
H3 is not needed at that level at all. That was one of my feedbacks when working in the HTML, please change it and apply the proper typography class

Reply:
Done